### PR TITLE
Auditissue

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
pls review

### CODE REVIEW

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -55,10 +55,11 @@ class Dashboard extends Component {
         data: [0, 0, 0, 0, 0, 0, 0]
       };
       this.props.campaignInfo.uniqueUsers.map(user => {
-        user.aggregations.users.buckets.map(bucket => {
+        console.log(user);
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: remove

File: src/components/WebsiteHome/WebsiteHome.js [hunk 2]
```diff
@@ -1,20 +1,15 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router';
 import Slider from 'react-slick';
+import NotificationView from '../Rules/NotificationView';
```
Comment (ShankyRana) [inline src/components/WebsiteHome/WebsiteHome.js]: use this Rules/NotificationView

File: src/components/WebsitePolicy/WebsitePolicy.js [hunk 3]
```diff
@@ -25,9 +26,16 @@ class WebsitePolicy extends Component {
     fetch('https://raw.githubusercontent.com/sagarshirbhate/Country-State-City-Database/master/Contries.json')
       .then(res => res.json())
       .then(res => this.setState({countryList : res.Countries}));
+    this.randomCodeGenerate();
+
+  }
+
+  randomCodeGenerate = () =>{
+    this.state.random = Math.floor(Math.random()*(+999999 - +111111) + +111111);
+    console.log('Random Number Generated : ' + this.state.random,'==========================' );
```
Comment (ShankyRana) [inline src/components/WebsitePolicy/WebsitePolicy.js]: remove

File: src/components/WebsitePolicy/WebsitePolicy.js [hunk 4]
```diff
@@ -73,14 +83,15 @@ class WebsitePolicy extends Component {
           text: 'Cancel',
           className: 'danger',
           action: function () {
-            Popup.close();
+            this.handleCountrySelect;
+            //Popup.close();
```
Comment (ShankyRana) [inline src/components/WebsitePolicy/WebsitePolicy.js]: remove

File: src/ducks/auth.js [hunk 5]
```diff
@@ -69,7 +76,8 @@ const initialState = fromJS({
   coupon: '',
   couponError: '',
   forgetError: '',
-  affiliateError: ''
+  affiliateError: '',
+  gdprformError:''
```
Comment (ShankyRana) [inline src/ducks/auth.js]: error are being set here but not displayed on the main component.

File: src/components/WebsiteHeader/WebsiteHeader.js [hunk 6]
```diff
@@ -5,7 +5,12 @@ import { LogoInfluence } from 'img';
 import  './WebsiteHeader.scss';
 class WebsiteHeader extends Component {
 
-
+  togglemenu = () => {
+    document.getElementById('hello').classList.toggle('customtoggleclass');
```
Comment (ShankyRana) [inline src/components/WebsiteHeader/WebsiteHeader.js]: can you use something better then this, React way....

File: src/components/WebsitePricing/WebsitePricing.scss [hunk 7]
```diff
@@ -120,10 +120,184 @@
...
       &:after {
-        margin-left: -.8em;
+        margin-left: .8em;
       }
     }
   }
 }
 
+}
+// @media only screen and (max-width: 1580px) {
```
Comment (ShankyRana) [inline src/components/WebsitePricing/WebsitePricing.scss]: remove if not needed

File: src/components/WebsiteHome/WebsiteHome.js (hunk 2 above)
Comment (Devilla) [reply src/components/WebsiteHome/WebsiteHome.js]: Ok.

Comment (Devilla) [reply src/components/WebsiteHome/WebsiteHome.js]: It's not working previous was ok. Module not found: Can't resolve 'Rules/NotificationView' in '/home/devilla/intergrations/frontend/src/components/WebsiteHome'

File: src/components/WebsitePolicy/WebsitePolicy.js (hunk 3 above)
Comment (Devilla) [reply src/components/WebsitePolicy/WebsitePolicy.js]: done.

File: src/components/WebsitePolicy/WebsitePolicy.js (hunk 4 above)
Comment (Devilla) [reply src/components/WebsitePolicy/WebsitePolicy.js]: done

File: src/ducks/auth.js (hunk 5 above)
Comment (Devilla) [reply src/ducks/auth.js]: ok.

File: src/components/WebsitePricing/WebsitePricing.scss (hunk 7 above)
Comment (Devilla) [reply src/components/WebsitePricing/WebsitePricing.scss]: done.

File: src/components/Dashboard/Dashboard.js (hunk 1 above)
Comment (Devilla) [reply src/components/Dashboard/Dashboard.js]: ok

File: src/components/WebsiteHeader/WebsiteHeader.js (hunk 6 above)
Comment (ShankyRana) [reply src/components/WebsiteHeader/WebsiteHeader.js]: is this done?

File: src/components/WebsiteHome/WebsiteHome.js (hunk 2 above)
Comment (Devilla) [reply src/components/WebsiteHome/WebsiteHome.js]: okay. Now i've made a different component for this thing.

File: src/components/WebsiteSignIn/WebsiteSignIn.scss [hunk 8]
```diff
@@ -13,16 +13,24 @@
...
           input {
             font-family: Verdana, Geneva, Tahoma, sans-serif;
             &::placeholder {
                 font-size: 14px !important;
               }
         }
+        .loginfrm {
+            margin-top: 0px;
+        }
+        .bg {
```
Comment (ShankyRana) [inline src/components/WebsiteSignIn/WebsiteSignIn.scss]: remove

File: src/components/WebsiteSignUp/WebsiteSignUp.scss [hunk 9]
```diff
@@ -28,6 +28,9 @@
         font-size: 14px !important;
       }
     }
+    .bg {
```
Comment (ShankyRana) [inline src/components/WebsiteSignUp/WebsiteSignUp.scss]: remove

File: src/components/WebsiteHeader/WebsiteHeader.js (hunk 6 above)
Comment (GAUTAMRAJU15) [reply src/components/WebsiteHeader/WebsiteHeader.js]: done

**Pull request closed without merging**